cmd/kubelet/app/server.go

```diff
@@ -117,7 +117,8 @@ func NewKubeletCommand() *cobra.Command {
 	kubeletConfig, err := options.NewKubeletConfiguration()
 	// programmer error
 	if err != nil {
-		klog.Fatal(err)
+		klog.ErrorS(err, "Failed create a new kubelet configuration")
+		os.Exit(255)
```

Comment:
Please use exit code 1 throughout this file, and not 255, to be consistent with the rest of the files in `cmd/`: https://cs.k8s.io/?q=os.Exit&i=nope&files=cmd%2F.*&repos=kubernetes/kubernetes

Reply:
Because the error code of `klog.Fatalf` will return 255, in order not to break compatibility, I returned 255

Reply:
I'll ask in Slack about it.

Reply:
we should not be using 255, but rather 1.

also idealy applications should have a single point of os.Exit() instead of multiple ones.

Reply:
> also idealy applications should have a single point of os.Exit() instead of multiple ones.

There are various reasons for calling `os.Exit(1)`. If only one `os.Exit()` is needed, do you have any good suggestions?

Reply:
deferring to the kubelet maintainers, but ideally errors should bubble up to a single point of os.Exit(x).